### DISCUSSION TITLE
Gauss Splatting

### DISCUSSION THREAD

**Discussion by alexLuky** [iTowns/itowns#2249] [Ideas]
There is a new 3D-rendering technology and it calls Gauss Splatting. It's alternative for photogrammetry 3D tiles. There is a [polycam](https://poly.cam/tools/gaussian-splatting) realization and it looks really great. Also there is a good version of three.js viewer adapter from [mkkellogg](https://github.com/mkkellogg/GaussianSplats3D) for even custom camera and renderer.

I think it would be gorgeous to integrate Gauss Splatting in Itowns. 

I've tried to integrate but faced with several issues like badly positioning. rotation on scene and render overlapping iTowns globe mesh and "Gauss Splats mesh".

![image](https://github.com/iTowns/itowns/assets/43720642/a2146f4e-0bbf-4ab3-99a8-5daf1447f091)

**Comment by Desplandis** [comment]
Hi @alexLuky!
As for now, it's not a short-term goal to integrate Gauss splatting into iTowns.
However, if you have an open implementation somewhere we could maybe help you with some of your issues.

If there is a other people interested in such feature, we could *maybe* add it to our long-term roadmap. Shall we convert this discussion into a feature proposal @jailln, @LoicUTC ?

P.S.: This reminds me that I shall read this research paper. x)

**Comment by LoicUTC** [comment]
Hello, thanks for the input.
3D Gauss splating is very interesting and promissing.
As @Desplandis said, don't hesitate to push your early implementation. We could help.

Otherwise, we could wait for the implementation into the master ThreeJS. 
But in any way, a feature proposal is a good process.
